src/SwipeProfiles.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import './Register.css';
+import {browserHistory} from 'react-router';
```

Comment:
not really sure about this one, but are you supposed to put `import { withRouter } from "react-router-dom";` there as well?